### PR TITLE
feat: add system priority class

### PR DESCRIPTION
**Pull request by rr-paras-patel**

### CODE REVIEW

File: edgedelta/templates/daemonset.yaml [hunk 1]
```diff
@@ -239,6 +239,7 @@ spec:
         {{- end }}
 
       terminationGracePeriodSeconds: 10
+      priorityClassName: {{ .Values.priorityClassName }}
```
Comment (taylanisikdemir) [inline edgedelta/templates/daemonset.yaml:242]: Hey @rr-paras-patel, thanks for taking the initiative and making these changes! 
This repo is updated by automation and we actually host the source of truth for our chart in our main monorepo.
@orhanasan will help getting this change published to new agent chart version tomorrow and let you know.

**Comment by rr-paras-patel** [comment]
You may also consider to add sha of config map or values.yaml as pod level so each helm upgrade will trigger rolling restart. and user don't need to do manually it.

**Comment by taylanisikdemir** [comment]
> You may also consider to add sha of configmap or values.yaml as pod abel so each helm upgrade will trigger rolling restart. and user don't need to do manually it. [helm/helm#9120 (comment)](https://github.com/helm/helm/issues/9120#issuecomment-743778301)

Good feedback. Most customers just use the config from API backend so our configmap based deployment options in helm is not super mature. We will look into this.

**Comment by taylanisikdemir** [comment]
Closing this PR because we will address this in our monorepo. Automation will push changes to this repo.

**Pull request closed without merging**